<launch>
    <!--node
        pkg="state_machine"
        type="test_pub.py"
        name="test_pub"
        output="screen"
    /-->
    <arg name="vesc1_ns" value="wheel_left"/>
    <arg name="vesc2_ns" value="wheel_right"/>
    <node pkg="state_machine" type="state_machine.py" name="state_machine" output="screen"/>
    <!--param name="port" value="$(arg vesc1_ns)" /-->
    <!--param name="port" value="$(arg vesc2_ns)" /-->

    <include file="$(find vesc_driver)/launch/vesc_driver_node.launch" ns="wheel_left">
		<arg name="node_name" value="vesc_wheel_l"/>
		<arg name="port" value="/dev/ttyACM0"/>
	</include>

	<include file="$(find vesc_driver)/launch/vesc_driver_node.launch" ns="wheel_right">
		<arg name="node_name" value="vesc_wheel_r"/>
		<arg name="port" value="/dev/ttyACM1"/>
	</include>

</launch> 